components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,1589 @@
++
+   \int_{A}\nabla_\perp \cdot \left( \tilde{h}\, \overline{{\bf u} \otimes {\bf u} }^{\tilde z} \right) dA
+& + \int_{A}\left[ {\bf u}\, \tilde{w}_{tr} \right]_{z=z^{\text{top}}} \, dA
+ - \int_{A}\left[ {\bf u}\, \tilde{w}_{tr} \right]_{z=z^{\text{bot}}} \, dA
+\\ & \; =
+- \int_{A} \tilde{h} \,\overline{  \mathbf{f} \times \mathbf{u} }^{\tilde z} \, dA
+- \int_{A} \tilde{h} \,\overline{  \nabla_\perp \Phi }^{\tilde z} \, dA
+- \int_{A} \frac{1}{\rho_0} \overline{\nabla_{\perp} p}^{z}  \, dA
++ \int_{A} \frac{1}{\rho_0} \overline{ \frac{\partial}{\partial x_i} \left( \tau_{ij} \right) }^z dA
+$$ (h-momentum-p-tau)
+
+Taking the limit as $A \rightarrow 0$, we arrive at the local, horizontally continuous form:
+
+$$
+\frac{\partial \tilde{h}\, \overline{  {\bf u} }^{\tilde z} }{\partial t} 
+& +
+   \nabla_\perp \cdot \left( \tilde{h}\, \overline{{\bf u} \otimes {\bf u} }^{\tilde z} \right) 
+ + \left[ {\bf u}\, \tilde{w}_{tr} \right]_{z=z^{\text{top}}} 
+ - \left[ {\bf u}\, \tilde{w}_{tr} \right]_{z=z^{\text{bot}}} 
+\\ & =
+-  \tilde{h} \,\overline{  \mathbf{f} \times \mathbf{u} }^{\tilde z} 
+-  \tilde{h} \,\overline{  \nabla_\perp \Phi }^{\tilde z} 
+-  \frac{1}{\rho_0} \overline{\nabla_{\perp} p}^{z}  
++  \frac{1}{\rho_0} \overline{ \frac{\partial}{\partial x_i} \left( \tau_{ij} \right) }^z 
+$$ (momentum-layered-differential-1)
+
+The first two terms are the material derivative, confined within the horizontal layer. Using the product rule, and layered mass conservation [](#layer-mass),
+
+$$
+\frac{\partial \tilde{h}\, \overline{  {\bf u} }^{\tilde z} }{\partial t} 
++ \nabla_\perp \cdot \left( \tilde{h}\, \overline{{\bf u} \otimes {\bf u} }^{\tilde z} \right) 
+&= 
+\frac{D_\perp \tilde{h}\, \overline{  {\bf u} }^{\tilde z} }{D t} \\
+&=
+\tilde{h}\frac{D_\perp  \overline{  {\bf u} }^{\tilde z} }{D t} 
++
+\overline{ {\bf u} }^{\tilde z} \frac{D_\perp \tilde{h}}{D t} \\
+&= 
+\tilde{h}\left(
+  \frac{\partial  \overline{  {\bf u} }^{\tilde z} }{\partial t} 
++ \nabla_\perp \cdot \left( \overline{{\bf u} \otimes {\bf u} }^{\tilde z} \right)  \right)
+- \overline{ {\bf u} }^{\tilde z} \left( 
+   \left[ {\tilde w}_{tr} \right]_{{\tilde z}={\tilde z}_k^{\text{top}}} 
+ - \left[ {\tilde w}_{tr} \right]_{{\tilde z}={\tilde z}_{k+1}^{\text{bot}}}
+\right) 
+$$ (2D-material-der-product)
+
+We now substitute [](#2D-material-der-product) into [](#momentum-layered-differential-1) and divide by $\tilde h$ to get
+
+$$
+ \frac{\partial \overline{  {\bf u} }^{\tilde z} }{\partial t} 
++ \nabla_\perp \cdot \left(  \overline{{\bf u} \otimes {\bf u} }^{\tilde z} \right) 
+& + \frac{\left[ {\bf u}\, \tilde{w}_{tr} \right]_{z=z^{\text{top}}} 
+ - \left[ {\bf u}\, \tilde{w}_{tr} \right]_{z=z^{\text{bot}}} }{\tilde h}
+-   \frac{\overline{ {\bf u} }^{\tilde z}}{\tilde{h}}  \left( 
+   \left[ {\tilde w}_{tr} \right]_{{\tilde z}={\tilde z}_k^{\text{top}}} 
+ - \left[ {\tilde w}_{tr} \right]_{{\tilde z}={\tilde z}_{k+1}^{\text{bot}}}\right)
```

Comment:
We need a sentence to explain these vertical transport terms, something like:
“These terms represent the net vertical transport of horizontal momentum across the layer boundaries and vanish when horizontal velocity is vertically uniform."